modules/custom/az_paragraphs/az_paragraphs_splitscreen/az_paragraphs_splitscreen.install

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/**
+ * @file
+ * Install, update and uninstall functions for az_paragraphs_splitscreen module.
+ */
+
+use Drupal\paragraphs\Entity\Paragraph;
+
+/**
+ * Add default splitscreen text width behavior setting.
+ */
+function az_paragraphs_splitscreen_update_1021301(&$sandbox) {
```

Comment:
If we're going to backport this change (include in a 2.12.x patch release), we should probably change number used in this update to reflect that.  We haven't always been super consistent about this in the past for backported DB updates but this is what our release policy suggests (although maybe not clearly enough): https://github.com/az-digital/az_quickstart/blob/main/RELEASES.md#update-hook-numbering-convention 

```suggestion
function az_paragraphs_splitscreen_update_1021201(&$sandbox) {
```